slow_odgi/flip.py

```diff
@@ -16,21 +17,55 @@ def path_is_rev(path, graph):
 
 
 def flip_path(path, graph):
-    """Flip the given path, if it is more reverse-oriented than forward."""
+    """Flip the given path if it is more reverse- than forward-oriented."""
     if path_is_rev(path, graph):
-        segments = []
+        path_segs = []
         for seg in reversed(path.segments):
-            segments.append (mygfa.Handle(seg.name, not seg.orientation))
-        return mygfa.Path(path.name+"_inv", segments, path.overlaps)
+            path_segs.append(mygfa.Handle(seg.name, not seg.orientation))
+        return mygfa.Path(f"{path.name}_inv", path_segs, path.overlaps)
     else:
         return path
 
 
+def dedup(list):
```

Comment:
It might be nice to annotate that this only works on lists of links:
```suggestion
def dedup(list: List[mygfa.Link]) -> List[mygfa.Link]:
```